src/cbor.c

```diff
@@ -139,27 +139,27 @@ size_t cbor_writeToken(uint8_t type, uint64_t value, uint8_t *buf, size_t bufSiz
 
 	if (value < VALUE_MIN_W1) {
 		CHECK_BUF_LEN(1);
-		u1be_write(buf, type | value);
+		u1be_write(inBuffer, type | value);
 		return 1;
 	} else if (value < VALUE_MIN_W2) {
 		CHECK_BUF_LEN(1 + 1);
-		u1be_write(buf, type | 24);
-		u1be_write(buf + 1, value);
+		u1be_write(inBuffer, type | 24);
+		u1be_write(inBuffer + 1, value);
 		return 1 + 1;
 	} else if (value < VALUE_MIN_W4) {
 		CHECK_BUF_LEN(1 + 2);
-		u1be_write(buf, type | 25);
-		u2be_write(buf + 1, value);
+		u1be_write(inBuffer, type | 25);
+		u2be_write(inBuffer + 1, value);
 		return 1 + 2;
 	} else if (value < VALUE_MIN_W8) {
 		CHECK_BUF_LEN(1 + 4);
-		u1be_write(buf, type | 26);
-		u4be_write(buf + 1, value);
+		u1be_write(inBuffer, type | 26);
+		u4be_write(inBuffer + 1, value);
 		return 1 + 4;
 	} else {
 		CHECK_BUF_LEN(1 + 8);
-		buf[0] = type | 27;
-		u8be_write(buf + 1, value);
+		inBuffer[0] = type | 27;
```

Comment:
hmm, can we change this to `u1be_write` in order to be consistent with the rest of the code?